ShellPkg/Library/UefiShellDebug1CommandsLib/HexEdit/MainHexEditor.c

```diff
@@ -115,7 +115,9 @@ HMainCommandDisplayHelp (
                               ,
                    NULL
                    );
-    ShellPrintEx (0, CurrentLine+1, L"%E%s%N", InfoString);
+    if (InfoString != NULL) {

```

Comment:
Should have an assert or handle the error case somehow I think

Reply:
I added an assert if InfoString is null. InfoString is only used here to call `ShellPrintEx` so it should be safe to continue.